CONTRIBUTING.rst

```diff
@@ -0,0 +1,480 @@
+   where code from all contributors is ultimately merged into and where
+   releases of the package will be made from.
+-  ``origin`` refers to the online fork you made of the upstream
+   ``drizzlepac`` repository.
+-  ``local`` refers to the clone you made of the origin on your
+   computer.
+-  The term ``remote`` in this context can refer to origin, or upstream.
+   in general, it means anything hosted online on Github.
+
+The first step is to create your own 'remote' (online) and 'local' (on
+your machine) clones of the central ``spacetelescope/drizzlepac``
+repository. You will make code changes on your machine to your 'local'
+clone, push these to 'origin' (your online fork), and finally, open a
+pull request to the ''main'' branch of ``spacetelescope/drizzlepac``.
+
+1. On the 'spacetelescope/drizzlepac' Github repository page, 'fork' the
+   DrizzlePac repository to your own account space by clicking the
+   appropriate button on the upper right-hand side. This will create an
+   online clone of the main DrizzlePac repository but instead of being
+   under the 'spacetelescope' organization, it will be under your
+   personal account space. It is necessary to fork the repository so
+   that many contributors aren't making branches directly on
+   'spacetelescope/drizzlepac'.
+
+2. Now that you have remotely forked ``drizzlepac``, it needs to be
+   downloaded to your machine. To create this 'local' clone, choose an
+   area on your file system and use the ``git clone`` command to dowload
+   your remote fork on to your machine.
+
+If you are using an SSH key to authenticate.
+
+   ::
+
+       >> cd directory
+       >> git clone git@github.com:<your_username>/drizzlepac.git
+
+Otherwise use the following. 
+
+   ::
+
+       >> cd directory
+       >> git clone https://github.com/<your_username>/drizzlepac.git
+
+3. Make sure that your references to 'origin' and 'upstream' are set
+   correctly - you will need this to keep everything in sync and push
+   your changes online. While your inital local clone will be an exact
+   copy of your remote, which is an exact copy of the 'upstream'
+   ``spacetelescope/drizzlepac``, these all must be kept in sync
+   manually (via git fetch/pull/push).
+
+   To check the current status of these references:
+
+   ::
+
+       >> git remote -v
+
+After your inital clone, you will likely be missing the reference to
+'upstream' (which is just the most commonly used name in git to refer to
+the main project repository - you can call this whatever you want but
+the origin/upstream conventions are most commonly used) - to set this,
+use the ``add`` git command:
+
+If you are using an SSH key to authenticate.
+
+::
+
+   >> git remote add upstream git@github.com:spacetelescope/drizzlepac.git
+
+Otherwise, you can simply set it to the repository URL but you will have
+to enter your password every time you fetch/push
+
+::
+
+   >> git remote add upstream https://github.com/spacetelescope/drizzlepac
```

Comment:
I am not sure this change is actually needed, but for completeness the URL would be:
https://github.com/spacetelescope/drizzlepac.git

Reply:
fixed.